web/src/views/web/page/inventory/join.ejs

```diff
@@ -36,8 +36,9 @@
 
         <p>This is the result of the following query:</p>
         <pre>
-  SELECT I.serial_num, I.container_id, I.type_name, I.weight, I.manufacture_date
-  FROM inventory I, storage_container S WHERE S.id = I.container_id AND S.id = < Given Container ID >;</pre>
+                SELECT I.serial_num, I.container_id, I.type_name, I.weight, I.manufacture_date
+                FROM inventory I, storage_container S
+                WHERE S.warehouse_id = < Given WarehouseID > AND I.container_id = S.id</pre>
```

Comment:
If you get rid of those extra spaces on the left it'll show properly on the page :)

Although if you put 0 spaces it looks like it's too close to the side lol so I've been putting 2 spaces. Tbh better thing to do is use 0 spaces and then fix the CSS but that's a problem for the future